galaxy_ng/app/settings.py

```diff
@@ -301,9 +303,10 @@
 GALAXY_DYNAMIC_SETTINGS = False
 
 # DJANGO ANSIBLE BASE RESOURCES REGISTRY SETTINGS
+ANSIBLE_BASE_RESOURCE_CONFIG_MODULE = "galaxy_ng.app.api.resource_api"
 ANSIBLE_BASE_ORGANIZATION_MODEL = "galaxy.Organization"
 ANSIBLE_BASE_JWT_VALIDATE_CERT = False
-ANSIBLE_BASE_JWT_KEY = "https://localhost"
+ANSIBLE_BASE_JWT_KEY = None  # This must be set in the environment.
```

Comment:
🚨 whenever `ANSIBLE_BASE_JWT_KEY` is set to a url such as `https://localhost` this will cause the hook `configure_resource_provider` to set API_HOSTNAME and CONTENT_HOSTNAME scheme and netloc to the same.

Which means this variable must be False whenever galaxy is running standalone (without a RESOURCE_SERVER) and to the https:// root whenever it is running behind the resource proxy.